content/sales.rst

```diff
@@ -0,0 +1,196 @@
+Sales and Marketing
+###################
+
+:menu_order: 008
+
+.. contents::
+
+
+Introduction
+============
+
+This page briefly describe operational Sales details.
+
+The main contact is Tim Adams, the main email group is at sales@proatria.com.
+
+Sales:
+
+* Provides the main software information to prospective customers, forwards
```

Comment:
Well... there is ``Sales/Support`` and then there is ``Sales/Marketing and Support`.

However once you start working on things like - creating banners, coming up with Google search campaigns, coming up with logos, working with designers for a page redesign then the activity becomes a ``Sales/Marketing`` activity and there is very little Support involved.

So with this page, I am seeing it as a ``Sales/Marketing`` combination.

Not sure if that makes sense.